x/tokenomics/types/params.go

```diff
@@ -201,3 +223,32 @@ func ValidateMintAllocationSum(params *Params) error {
 
 	return nil
 }
+
+// ValidateDaoRewardAddress validates the DaoRewardAddress param.
+func ValidateDaoRewardAddress(daoRewardAddress any) error {
+	daoRewardAddressStr, ok := daoRewardAddress.(string)
+	if !ok {
+		return ErrTokenomicsParamInvalid.Wrapf("invalid parameter type: %T", daoRewardAddress)
+	}
+
+	// DEV_NOTE: Empty string is valid if mint_allocation_dao is 0.
```

Comment:
Hmmm, why?

Any chance we can just prevent this from ever being zero altogether and avoid bugs or special/edge cases?